pkg/recon/store.go

```
@@ -1,6 +1,7 @@
 package recon
 
 import (
+	"github.com/praetorian-inc/augustus/internal/observed"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)